### PR TITLE
Feature/deepspeech distributed benchmarks

### PR DESCRIPTION
**Pull request by roscisz**

### CODE REVIEW

File: examples/deepspeech/README.md [hunk 1]
```diff
@@ -96,27 +97,55 @@ For example, to use GPUs 1 and 2, set CUDA_VISIBLE_DEVICES=1,2 and to use all GP
...
 ### MultiGPU scalability
 
 The following results show performance results on NVIDIA® DGX Station™, depending on the choice utilized GPUs. The
 results are marked with ID's of the used GPUs, for example '013' means that CUDA_VISIBLE_DEVICES was set to 0,1,3.
 
-![multigpu_128](https://raw.githubusercontent.com/roscisz/TensorHive/develop/examples/deepspeech/img/multigpu_128.png)
-![multigpu_64](https://raw.githubusercontent.com/roscisz/TensorHive/develop/examples/deepspeech/img/multigpu_64.png)
+![multigpu_128](https://raw.githubusercontent.com/roscisz/TensorHive/master/examples/deepspeech/img/multigpu_128.png)
+
+Interestingly, in the cases of utilizing two GPUs, it is significant which exactly GPUs are used exactly. For example,
```
Comment (PiotrowskiD) [inline examples/deepspeech/README.md]: which ~~exactly~~ GPUs are used exactly*

File: examples/deepspeech/README.md [hunk 2]
```diff
@@ -96,27 +97,55 @@ For example, to use GPUs 1 and 2, set CUDA_VISIBLE_DEVICES=1,2 and to use all GP
...
+parameter server and worker processes. The 'bin/run-cluster.sh' script is helpful for configuring and running
+these processes on a single machine. The script accepts an argument in a p:w:g format, where p denotes the 
+number of used parameter servers, w denotes the number of worker processes and g denotes the number of
+GPUs used by individual worker processes.
+
+For example, running our benchmark on the distributed training
+application using 1 parameter server and 4 workers using 1 GPU each would require executing the following command:
+
+```bash
+LD_LIBRARY_PATH=native_client/ bin/run-cluster.sh 1:4:1 --script="python3 DeepSpeech.py" --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --train_batch_size=64 --epoch=1000 --benchmark_warmup_steps=10 --benchmark_steps=10 --log_level=3
```
Comment (PiotrowskiD) [inline examples/deepspeech/README.md]: + --noshow_progressbar as it is currently not working with distributed training

File: examples/deepspeech/README.md (hunk 1 above)
Comment (roscisz) [reply examples/deepspeech/README.md]:
```suggestion
Interestingly, in the cases of utilizing two GPUs, it is significant which GPUs are used exactly. For example,
```

File: examples/deepspeech/README.md (hunk 2 above)
Comment (roscisz) [reply examples/deepspeech/README.md]: Isn't --log_level=3 enough?

Comment (roscisz) [reply examples/deepspeech/README.md]:
```suggestion
LD_LIBRARY_PATH=native_client/ bin/run-cluster.sh 1:4:1 --script="python3 DeepSpeech.py" --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --train_batch_size=64 --epoch=1000 --benchmark_warmup_steps=10 --benchmark_steps=10 --log_level=3 --noshow_progressbar
```

Comment (PiotrowskiD) [reply examples/deepspeech/README.md]: > Isn't --log_level=3 enough?

unofrtunately no

**Pull request merged**